eskip/string.go

```diff
@@ -78,7 +80,7 @@ func argsString(args []interface{}) string {
 	return strings.Join(sargs, ", ")
 }
 
-func (r *Route) predicateString() string {
+func (r *Route) predicateString(prettyPrintInfo PrettyPrintInfo) string {
```

Comment:
We do not need whole PrettyPrintInfo here (it is also passed by value - creates a copy), lets just pass bool.